### ISSUE TITLE
Input element focus doesn't work

### ISSUE THREAD

**Issue by muhammadumairaslam** [fullcalendar/fullcalendar-scheduler#548]
Hi,

Just used input element instead of button as you explaind in the following comment.
https://github.com/fullcalendar/fullcalendar-scheduler/issues/508#issuecomment-481034843

https://codepen.io/muhammadumairaslam/pen/PvZMer

Thanks

**Comment by acerix** [comment]
I think it's not considered a bug because `resourceLabelText` is not intended to have HTML in it, and I'm not sure what the cause.  You could manually focus it onclick like this: https://codepen.io/acerix/pen/ZdbevP?editors=0010

**Comment by muhammadumairaslam** [comment]
Hi @acerix,
thanks for your response.

but it's taking a long time to focus while adding more resources. 
right click event is working perfectly instead of click.

https://codepen.io/muhammadumairaslam/pen/PvZMer
please open in chrome browser to check.

**Comment by acerix** [comment]
I can see there's a delay after clicking the resource column in Chrome when there are many resources, but didn't find an obvious solution so that should be posted as a separate issue. There is a related issue to improve performance:

https://github.com/fullcalendar/fullcalendar/issues/2524

**Comment by muhammadumairaslam** [comment]
as per my finding that issue occurred due to register too many `mousedown` events on calendar.  see snapshot for reference, 

![aa](https://i.ibb.co/cXyTHXR/eventlistener.png)

















if remove the interaction plugin or remove all `mousedown` events so it will work fine.

i didn't know why resource area is calling the [event drag and resize](https://fullcalendar.io/docs/event-dragging-resizing) function.

i just add condition in `handleMouseDown` function to stop unnecessary calls which has defined in [here](https://github.com/fullcalendar/fullcalendar/blob/master/src/interaction/dnd/PointerDragging.ts). 

```
this.handleMouseDown = function (ev) {
     if (!core.elementClosest(ev.target, '.disable-dragging') &&
```

**Comment by arshaw** [comment]
I agree, fullcalendar is too aggressive with disallowing text selection. This was originally implemented to prevent unwanted text selection while event dragging/resizing, but for some reason it's also applied to almost every element. will fix eventually
